webcompat/views.py

```diff
@@ -274,7 +275,16 @@ def contributors():
 @app.route('/tools/cssfixme')
 def cssfixme():
     '''Route for CSS Fix me tool'''
-    return render_template('cssfixme.html')
+
+    url = request.args.get('url')
+    escaped_csscode = ''
+    if url:
+        req = requests.get(url)
```

Comment:
The cool thing is that the message in the console is quite interesting.

```
  File "/Users/karl/.virtualenvs/webcompatcom/lib/python2.7/site-packages/requests/models.py", line 338, in prepare_url
    "Perhaps you meant http://{0}?".format(url))
MissingSchema: Invalid URL u'blah': No schema supplied. Perhaps you meant http://blah?
```

MissingSchema.